public/app/features/datasources/state/actions.ts

```diff
@@ -118,9 +120,15 @@ export function loadDataSource(uid: string): ThunkResult<void> {
     const dataSource = await getDataSourceUsingUidOrId(uid);
     const pluginInfo = (await getPluginSettings(dataSource.type)) as DataSourcePluginMeta;
     const plugin = await importDataSourcePlugin(pluginInfo);
-
+    const isBackend = plugin.DataSourceClass.prototype instanceof DataSourceWithBackend;
+    const meta = {
+      ...pluginInfo,
+      backend: isBackend,
```

Comment:
```suggestion
      isBackend: isBackend,
```
Maybe the prop should be named `isBackend` as well?